compute_tools/src/bin/compute_ctl.rs

```diff
@@ -285,11 +292,11 @@ fn main() -> Result<()> {
     exit(exit_code.unwrap_or(1))
 }
 
+project_git_version!(GIT_VERSION);
```

Comment:
I saw `GIT_VERSION` in `Dockerfile`, and should we use the env variable instead of retrieving from the workspace here?

---

I'm generally not in favor of such git version related build macros because everything I commit the changes, cargo will need to do a recompile even if nothing is actually changed 🤣 But it's up to you whether to add it.

Reply:
I just reverted this part.